main/main.go

```diff
@@ -110,15 +111,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	// We want to remove logr.Logger first step to move to logrus
+	// in the future we will have only one of them
+	newLogger := logrus.New()
 	logr.SetFormatter(&logrus.TextFormatter{
 		DisableColors:   true,
 		TimestampFormat: "2006-01-02 15:04:05",
 		FullTimestamp:   true,
 	})
-	op := ops.NewOps(logr, ops.NewExecutor(logr, true))
-	rpmOstreeClient := rpmostreeclient.NewClient("ibu-imager", op)
-
 	log := ctrl.Log.WithName("controllers").WithName("ImageBasedUpgrade")
+
+	executor := ops.NewChrootExecutor(newLogger, true, utils.Host)
+	rpmOstreeClient := rpmostreeclient.NewClient("ibu-imager", executor)
```

Comment:
```suggestion
	rpmOstreeClient := rpmostreeclient.NewClient("ibu-controller", executor)
```

Reply:
This client.id is not used anywhere. It is just an extra variable.

Reply:
@donpenney this client is assigned to the ibu reconciler struct